app/models/cloud_volume.rb

```diff
@@ -54,14 +54,6 @@ def self.create_volume(ems_id, options = {})
     tenant = options[:cloud_tenant]
 
     created_volume = klass.raw_create_volume(ext_management_system, options)
-
```

Comment:
@aufi is any code relying on that returning the actual `CloudVolume` object?  I'm not seeing anything other than the spec test.